tests/settings.py

```diff
@@ -23,10 +27,19 @@
 
 SECRET_KEY = '0'
 
+if django.VERSION < (1, 11):
+    template_directories = []
+else:
+    template_directories = [
+        os.path.join(
+            os.path.dirname(django.__file__),
+            "forms/templates/"
+        )
+    ]
```

Comment:
this code block is necessary for finding django 1.11's form templates. Though for the most part they aren't used, a couple widgets rely on their base widgets for rendering by default still.